vllm/worker/model_runner.py

```diff
@@ -539,12 +543,13 @@ def _make_tensor_with_pad(
     pad: int,
     dtype: torch.dtype,
     device: Union[str, torch.device] = "cuda",
+    pin_memory=False
 ) -> torch.Tensor:
     padded_x = [_pad_to_max(x_i, max_len, pad) for x_i in x]
     return torch.tensor(padded_x,
                         dtype=dtype,
                         device=device,
-                        pin_memory=device == 'cpu')
+                        pin_memory=pin_memory)
```

Comment:
```suggestion
                        pin_memory=pin_memory and str(device) == "cpu")
```

Reply:
I guess a extra bit of safety is always welcomed

Reply:
I guess this is needed because `device` can be "cuda"?